devops/ansible/ssl.yml

```diff
@@ -0,0 +1,39 @@
+- name: Install certbot
+  apt:
+    name: certbot
+    update_cache: yes
+  become: true
+
+- name: Copy post cert renewal hook script
+  copy:
+    src: arango-ssl.sh
+    dest: /etc/letsencrypt/renewal-hooks/deploy/
+    mode: u=rwx,g=,o=
+  become: true
+
+- name: Set server name inside of the post cert renewal hook script
+  replace:
+    path: /etc/letsencrypt/renewal-hooks/deploy/arango-ssl.sh
+    regexp: '(SERVER_NAME)'
+    replace: '{{ server_name }}'
+  become: true
+
+- name: Generate a certificate
+  shell:
+    cmd: certbot certonly -d {{ server_name }} --standalone -m jwilburn@sci.utah.edu --agree-tos -n
+  become: true
+
+- name: Run arango-ssl.sh (doesn't run above since it's not a renewal)
```

Comment:
I don't understand what the parenthesis means.

Reply:
I add the script for making the server.pem file above, but it doesn't run when the cert is created, only when the cert is renewed. I was just saying that we have to manually trigger it, even though it seems like it should run on the hook